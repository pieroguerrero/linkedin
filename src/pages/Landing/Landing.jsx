import React from "react";
import { Hero } from "./components/Hero";
import { LandingHeader } from "./components/LandingHeader";
import { Section } from "./components/Section";
import { RoundedTextButton } from "../../components/form-controls";
import { showNotAvailableToast } from "../../utilities";

const Landing = () => {
  const handleGeneralClick = () => {
    showNotAvailableToast();
  };

  return (
    <div className="flex flex-col min-h-screen  items-center">
      <LandingHeader />
      <main className="flex flex-col max-w-6xl w-full overflow-hidden">
        {/* //TODO:Remove the max-w-6xl from the main component and include it in every subcomponent */}
        <Hero />
        <Section
          strTitle="Explore topics you are interested in"
          strSubTitle="CONTENT TOPICS"
          strColor="ligth-gray"
        >
          <>
            <RoundedTextButton
              handleClick={handleGeneralClick}
              strColor="blue"
              strText="See All Topics"
              booBorder={true}
              booColoredBackground={false}
              booFullWidth={false}
            ></RoundedTextButton>
            <RoundedTextButton
              handleClick={handleGeneralClick}
              strColor="gray"
              strText="Remote"
              booBorder={true}
              booColoredBackground={false}
              booFullWidth={false}
            ></RoundedTextButton>
            <RoundedTextButton
              handleClick={handleGeneralClick}
              strColor="gray"
              strText="Work from Home"
              booBorder={true}
              booColoredBackground={false}
              booFullWidth={false}
            ></RoundedTextButton>
            <RoundedTextButton
              handleClick={handleGeneralClick}
              strColor="gray"
              strText="Retirement"
              booBorder={true}
              booColoredBackground={false}
              booFullWidth={false}
            ></RoundedTextButton>
            <RoundedTextButton
              handleClick={handleGeneralClick}
              strColor="gray"
              strText="Internships"
              booBorder={true}
              booColoredBackground={false}
              booFullWidth={false}
            ></RoundedTextButton>
            <RoundedTextButton
              handleClick={handleGeneralClick}
              strColor="gray"
              strText="Freelancer"
              booBorder={true}
              booColoredBackground={false}
              booFullWidth={false}
            ></RoundedTextButton>
            <RoundedTextButton
              handleClick={handleGeneralClick}
              strColor="gray"
              strText="Salary and Compensation"
              booBorder={true}
              booColoredBackground={false}
              booFullWidth={false}
            ></RoundedTextButton>
            <RoundedTextButton
              handleClick={handleGeneralClick}
              strColor="gray"
              strText="Starting a job"
              booBorder={true}
              booColoredBackground={false}
              booFullWidth={false}
            ></RoundedTextButton>
          </>
        </Section>
      </main>
    </div>
  );
};

export { Landing };
