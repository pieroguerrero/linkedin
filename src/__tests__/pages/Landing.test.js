import { render } from "@testing-library/react";
import { Landing } from "../../pages/Landing/Landing";

describe("Landing page Snapshots", () => {
  it("Desktop", () => {
    const { container } = render(<Landing />);
    expect(container).toMatchSnapshot();
  });
});
