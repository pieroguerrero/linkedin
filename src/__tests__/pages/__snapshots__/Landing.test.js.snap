// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing page Snapshots Desktop 1`] = `
<div>
  <div
    class="flex flex-col min-h-screen  items-center"
  >
    <header
      class="w-full flex justify-center"
    >
      <nav
        class="flex-1 pt-3 pb-2 flex items-center justify-between relative flex-nowrap w-full max-w-6xl px-4"
      >
        <div
          class="sm:hidden"
        >
          <button
            aria-expanded="false"
            class="flex items-center justify-center gap-1 cursor-pointer"
            type="button"
          >
            <span
              aria-hidden="true"
              class="flex items-center h-3 w-3 text-color-blue mr-1"
            >
              <svg
                fill="none"
                focusable="false"
                height="8"
                width="14"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M.333 4.667h13.334V3.333H.333v1.334Zm0-3.334h13.334V0H.333v1.333ZM.333 8h13.334V6.667H.333V8Z"
                  fill="currentColor"
                  fill-rule="evenodd"
                />
              </svg>
            </span>
            <span
              class="font-sans text-sm font-bold text-color-blue leading-[1.2]"
            >
              Menu
            </span>
          </button>
        </div>
        <div>
          <svg
            class="w-[84px] h-[21px] sm:w-[135px] sm:h-[34px] text-color-blue "
            focusable="false"
            preserveAspectRatio="xMinYMin meet"
            version="1.1"
            viewBox="0 0 84 21"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="none"
              fill-rule="evenodd"
              stroke="none"
              stroke-width="1"
            >
              <path
                d="M19.479,0 L1.583,0 C0.727,0 0,0.677 0,1.511 L0,19.488 C0,20.323 0.477,21 1.333,21 L19.229,21 C20.086,21 21,20.323 21,19.488 L21,1.511 C21,0.677 20.336,0 19.479,0"
                transform="translate(63.000000, 0.000000)"
              />
              <path
                d="M82.479,0 L64.583,0 C63.727,0 63,0.677 63,1.511 L63,19.488 C63,20.323 63.477,21 64.333,21 L82.229,21 C83.086,21 84,20.323 84,19.488 L84,1.511 C84,0.677 83.336,0 82.479,0 Z M71,8 L73.827,8 L73.827,9.441 L73.858,9.441 C74.289,8.664 75.562,7.875 77.136,7.875 C80.157,7.875 81,9.479 81,12.45 L81,18 L78,18 L78,12.997 C78,11.667 77.469,10.5 76.227,10.5 C74.719,10.5 74,11.521 74,13.197 L74,18 L71,18 L71,8 Z M66,18 L69,18 L69,8 L66,8 L66,18 Z M69.375,4.5 C69.375,5.536 68.536,6.375 67.5,6.375 C66.464,6.375 65.625,5.536 65.625,4.5 C65.625,3.464 66.464,2.625 67.5,2.625 C68.536,2.625 69.375,3.464 69.375,4.5 Z"
                fill="currentColor"
              />
            </g>
            <g>
              <path
                d="M60,18 L57.2,18 L57.2,16.809 L57.17,16.809 C56.547,17.531 55.465,18.125 53.631,18.125 C51.131,18.125 48.978,16.244 48.978,13.011 C48.978,9.931 51.1,7.875 53.725,7.875 C55.35,7.875 56.359,8.453 56.97,9.191 L57,9.191 L57,3 L60,3 L60,18 Z M54.479,10.125 C52.764,10.125 51.8,11.348 51.8,12.974 C51.8,14.601 52.764,15.875 54.479,15.875 C56.196,15.875 57.2,14.634 57.2,12.974 C57.2,11.268 56.196,10.125 54.479,10.125 L54.479,10.125 Z"
                fill="currentColor"
              />
              <path
                d="M47.6611,16.3889 C46.9531,17.3059 45.4951,18.1249 43.1411,18.1249 C40.0001,18.1249 38.0001,16.0459 38.0001,12.7779 C38.0001,9.8749 39.8121,7.8749 43.2291,7.8749 C46.1801,7.8749 48.0001,9.8129 48.0001,13.2219 C48.0001,13.5629 47.9451,13.8999 47.9451,13.8999 L40.8311,13.8999 L40.8481,14.2089 C41.0451,15.0709 41.6961,16.1249 43.1901,16.1249 C44.4941,16.1249 45.3881,15.4239 45.7921,14.8749 L47.6611,16.3889 Z M45.1131,11.9999 C45.1331,10.9449 44.3591,9.8749 43.1391,9.8749 C41.6871,9.8749 40.9121,11.0089 40.8311,11.9999 L45.1131,11.9999 Z"
                fill="currentColor"
              />
              <polygon
                fill="currentColor"
                points="38 8 34.5 8 31 12 31 3 28 3 28 18 31 18 31 13 34.699 18 38.241 18 34 12.533"
              />
              <path
                d="M16,8 L18.827,8 L18.827,9.441 L18.858,9.441 C19.289,8.664 20.562,7.875 22.136,7.875 C25.157,7.875 26,9.792 26,12.45 L26,18 L23,18 L23,12.997 C23,11.525 22.469,10.5 21.227,10.5 C19.719,10.5 19,11.694 19,13.197 L19,18 L16,18 L16,8 Z"
                fill="currentColor"
              />
              <path
                d="M11,18 L14,18 L14,8 L11,8 L11,18 Z M12.501,6.3 C13.495,6.3 14.3,5.494 14.3,4.5 C14.3,3.506 13.495,2.7 12.501,2.7 C11.508,2.7 10.7,3.506 10.7,4.5 C10.7,5.494 11.508,6.3 12.501,6.3 Z"
                fill="currentColor"
              />
              <polygon
                fill="currentColor"
                points="3 3 0 3 0 18 9 18 9 15 3 15"
              />
            </g>
          </svg>
        </div>
        <div
          class="hidden sm:flex sm:gap-1"
        >
          <div
            class="flex items-center justify-center"
          >
            <ul
              class="flex flex-col sm:flex-row gap-0.5 sm:gap-1 sm:items-center w-full"
            >
              <li>
                <button
                  class="flex items-center flex-row sm:flex-col gap-1.5 sm:gap-0 h-[52px] text-color-text-low-emphasis hover:text-black w-fit sm:w-[80px] justify-center"
                  type="button"
                >
                  <span
                    class="flex w-8 sm:w-auto sm:h-5 flex-shrink-0"
                  >
                    <svg
                      fill="none"
                      focusable="false"
                      height="20"
                      width="21"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M18.07 2.93A10 10 0 1 0 3.926 17.07 10 10 0 0 0 18.07 2.93ZM5.34 15.66A8 8 0 1 1 16.66 4.353 8 8 0 0 1 5.34 15.66ZM9 8l-3 7a7 7 0 1 1 9.9-9.9L9 8Zm1.12 2.88a1.24 1.24 0 1 1 1.76-1.747 1.24 1.24 0 0 1-1.76 1.747ZM15.95 15A7 7 0 0 1 6 15l7-3 2.95-6.95a7 7 0 0 1 0 9.95Z"
                        fill="currentColor"
                      />
                    </svg>
                  </span>
                  <span
                    class="text-sm font-bold sm:font-normal break-all"
                  >
                    Discover
                  </span>
                </button>
              </li>
              <li>
                <button
                  class="flex items-center flex-row sm:flex-col gap-1.5 sm:gap-0 h-[52px] text-color-text-low-emphasis hover:text-black w-fit sm:w-[80px] justify-center"
                  type="button"
                >
                  <span
                    class="flex w-8 sm:w-auto sm:h-5 flex-shrink-0"
                  >
                    <svg
                      fill="none"
                      focusable="false"
                      height="20"
                      width="18"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M9 14v6H0v-6c0-1.7 1.3-3 3-3h3c1.7 0 3 1.3 3 3Zm5.5-3c1.9 0 3.5-1.6 3.5-3.5S16.4 4 14.5 4 11 5.6 11 7.5s1.6 3.5 3.5 3.5Zm1 2h-2c-1.4 0-2.5 1.1-2.5 2.5V20h7v-4.5c0-1.4-1.1-2.5-2.5-2.5ZM4.5 0C2 0 0 2 0 4.5S2 9 4.5 9 9 7 9 4.5 7 0 4.5 0Z"
                        fill="currentColor"
                      />
                    </svg>
                  </span>
                  <span
                    class="text-sm font-bold sm:font-normal break-all"
                  >
                    People
                  </span>
                </button>
              </li>
              <li>
                <button
                  class="flex items-center flex-row sm:flex-col gap-1.5 sm:gap-0 h-[52px] text-color-text-low-emphasis hover:text-black w-fit sm:w-[80px] justify-center"
                  type="button"
                >
                  <span
                    class="flex w-8 sm:w-auto sm:h-5 flex-shrink-0"
                  >
                    <svg
                      fill="none"
                      focusable="false"
                      height="24"
                      width="24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clip-rule="evenodd"
                        d="M23 3H1a1 1 0 0 0-1 1v16a1 1 0 0 0 1 1h22a1 1 0 0 0 1-1V4a1 1 0 0 0-1-1ZM2 19h20V5H2v14Z"
                        fill="currentColor"
                        fill-rule="evenodd"
                      />
                      <path
                        clip-rule="evenodd"
                        d="M4 9h6V7H4v2Zm0 4h6v-2H4v2Zm0 4h6v-2H4v2Zm-2 2h10V5H2v14Z"
                        fill="currentColor"
                        fill-opacity=".25"
                        fill-rule="evenodd"
                      />
                      <path
                        clip-rule="evenodd"
                        d="M14 9h6V7h-6v2Zm0 4h6v-2h-6v2Zm6 4h-6v-2h6v2Z"
                        fill="currentColor"
                        fill-opacity=".6"
                        fill-rule="evenodd"
                      />
                      <path
                        clip-rule="evenodd"
                        d="M10 7.534v8.933a.28.28 0 0 0 .439.23l6.433-4.436A.307.307 0 0 0 17 12a.305.305 0 0 0-.128-.26l-6.433-4.437a.28.28 0 0 0-.439.23Z"
                        fill="currentColor"
                        fill-rule="evenodd"
                      />
                    </svg>
                  </span>
                  <span
                    class="text-sm font-bold sm:font-normal break-all"
                  >
                    Learning
                  </span>
                </button>
              </li>
              <li>
                <button
                  class="flex items-center flex-row sm:flex-col gap-1.5 sm:gap-0 h-[52px] text-color-text-low-emphasis hover:text-black w-fit sm:w-[80px] justify-center"
                  type="button"
                >
                  <span
                    class="flex w-8 sm:w-auto sm:h-5 flex-shrink-0"
                  >
                    <svg
                      fill="none"
                      focusable="false"
                      height="18"
                      width="20"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M15 4V3c0-1.7-1.3-3-3-3H8C6.3 0 5 1.3 5 3v1H0v4c0 1.7 1.3 3 3 3h14c1.7 0 3-1.3 3-3V4h-5ZM7 3c0-.6.4-1 1-1h4c.6 0 1 .4 1 1v1H7V3Zm10 9c1.2 0 2.3-.5 3-1.4V15c0 1.7-1.3 3-3 3H3c-1.7 0-3-1.3-3-3v-4.4c.7.9 1.8 1.4 3 1.4h14Z"
                        fill="currentColor"
                      />
                    </svg>
                  </span>
                  <span
                    class="text-sm font-bold sm:font-normal break-all"
                  >
                    Jobs
                  </span>
                </button>
              </li>
            </ul>
            <span
              class=" border-r border-color-gray-medium border-solid h-9"
            />
          </div>
          <button
            class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
            type="button"
          >
            Join now
          </button>
          <button
            class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-blue hover:bg-color-button-blue text-color-button-blue"
            type="button"
          >
            Sign in
          </button>
        </div>
        <div
          class="flex items-baseline justify-center sm:hidden"
        >
          <button
            class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-blue hover:bg-color-button-blue text-color-button-blue"
            type="button"
          >
            Join now
          </button>
        </div>
      </nav>
    </header>
    <main
      class="flex flex-col items-center w-full overflow-hidden"
    >
      <section
        class="flex flex-col w-full max-w-6xl sm:flex-row px-4 sm:pt-[47px] pt-[24px]"
      >
        <div
          class=" sm:w-[55%]"
        >
          <form
            class="w-full sm:w-[408px]"
          >
            <p
              class="pb-[24px] sm:pb-0 text-[33px] sm:text-[58px] text-color-text-accent sm:font-light leading-[1.2]  sm:text-opacity-90"
            >
              Welcome to your professional community
            </p>
            <div
              class=" pt-3 pb-7 sm:pt-8 sm:pb-4"
            >
              <div
                class="flex flex-col gap-3"
              >
                <input
                  class="w-full h-12 border-color-button-gray rounded-sm bg-transparent px-3 text-base"
                  id="input-text-email"
                  name="input-text-email"
                  placeholder="Email or phone number"
                  type="email"
                />
                <input
                  id="input-text-pass"
                  name="input-text-pass"
                  placeholder="Password (8+ characters)"
                  type="password"
                />
              </div>
              <p
                class=" py-3 text-[15px] leading-5 text-color-text font-medium"
              >
                By clicking Agree & Join, you agree to the LinkedIn
                 
                <span
                  class=" hover:cursor-pointer hover:underline text-color-blue font-bold"
                >
                  User Agreement, Privacy Policy
                </span>
                 and 
                <span
                  class="hover:cursor-pointer hover:underline text-color-blue  font-bold"
                >
                  Cookie Policy
                </span>
                .
              </p>
              <div
                class="sm:text-xl"
              >
                <button
                  class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px]  w-full border border-solid  border-color-button-blue bg-color-button-blue hover:bg-color-button-blue-darker text-white"
                  type="button"
                >
                  Agree & Join
                </button>
              </div>
              <div
                class=" mt-4"
              >
                <button
                  class="flex justify-center items-center py-3 px-6 font-extrabold rounded-[28px] w-full border border-solid border-color-button-gray border-opacity-60 hover:border-opacity-90 hover:bg-black hover:bg-opacity-[0.04]"
                  type="button"
                >
                  <div
                    class="flex gap-4 sm:gap-6 items-center justify-center"
                  >
                    <svg
                      focusable="false"
                      height="24px"
                      preserveAspectRatio="xMinYMin meet"
                      viewBox="0 0 24 24"
                      width="24px"
                      x="0"
                      xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink"
                      y="0"
                    >
                      <g>
                        <path
                          d="M12.1,5.8c1.6-0.1,3.1,0.5,4.3,1.6l2.6-2.7c-1.9-1.8-4.4-2.7-6.9-2.7c-3.8,0-7.2,2-9,5.3l3,2.4C7.1,7.2,9.5,5.7,12.1,5.8z"
                          fill="#E94435"
                        />
                        <path
                          d="M5.8,12c0-0.8,0.1-1.6,0.4-2.3l-3-2.4C2.4,8.7,2,10.4,2,12c0,1.6,0.4,3.3,1.1,4.7l3.1-2.4C5.9,13.6,5.8,12.8,5.8,12z"
                          fill="#F8BB15"
                        />
                        <path
                          d="M15.8,17.3c-1.2,0.6-2.5,1-3.8,0.9c-2.6,0-4.9-1.5-5.8-3.9l-3.1,2.4C4.9,20,8.3,22.1,12,22c2.5,0.1,4.9-0.8,6.8-2.3L15.8,17.3z"
                          fill="#34A751"
                        />
                        <path
                          d="M22,12c0-0.7-0.1-1.3-0.2-2H12v4h6.1v0.2c-0.3,1.3-1.1,2.4-2.2,3.1l3,2.4C21,17.7,22.1,14.9,22,12z"
                          fill="#547DBE"
                        />
                      </g>
                    </svg>
                    <p
                      class=" text-color-button-gray text-lg sm:text-xl font-medium"
                    >
                      Join with Google
                    </p>
                  </div>
                </button>
              </div>
              <div
                class="flex items-center justify-center my-4 px-4  before:h-[1px] before:w-auto before:flex-1 before:bg-color-button-gray before:bg-opacity-50 before:content-[''] after:h-[1px] after:w-auto after:flex-1 after:bg-color-button-gray after:content-['']  after:bg-opacity-50"
              >
                <span
                  class="text-color-text-low-emphasis px-2"
                >
                  or
                </span>
              </div>
              <div
                class=" mt-4"
              >
                <button
                  class="flex justify-center items-center py-3 px-6 font-extrabold rounded-[28px] w-full border border-solid border-color-button-gray border-opacity-60 hover:border-opacity-90 hover:bg-black hover:bg-opacity-[0.04]"
                  type="button"
                >
                  <div
                    class="flex gap-6 items-center justify-center"
                  >
                    <p
                      class=" text-color-text text-lg sm:text-[21px] font-bold"
                    >
                      Continue as a 
                      <strong
                        class=" text-color-blue"
                      >
                        Guest
                      </strong>
                    </p>
                  </div>
                </button>
              </div>
            </div>
          </form>
        </div>
        <div
          class="flex"
        >
          <img
            alt="Welcome to your professional community"
            class="flip-rtl block sm:w-[700px] sm:h-[560px] sm:relative flex-shrink w-[374px] h-[214px] static"
            src="heroImage.svg"
          />
        </div>
      </section>
      <section
        class="w-full flex justify-center bg-[#F3F2F0]"
      >
        <div
          class="flex flex-col gap-8 sm:gap-[72px] max-w-6xl flex-nowrap  sm:flex-row w-full py-14 sm:py-[60px] px-4"
        >
          <p
            class=" mb-0 text-[34px] leading-10 sm:w-[408px] sm:leading-[60px] sm:text-[50px] font-normal sm:font-light text-color-text-darker"
          >
            Explore topics you are interested in
          </p>
          <div
            class="flex flex-col gap-4 max-w-[343px] sm:max-w-[640px]"
          >
            <p
              class=" uppercase text-color-text font-bold text-[17px]"
            >
              CONTENT TOPICS
            </p>
            <div
              class="flex flex-wrap gap-x-[6px] gap-y-3 text-[19px] sm:text-[21px] font-bold"
            >
              <button
                class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-blue hover:bg-color-button-blue text-color-button-blue"
                type="button"
              >
                See All Topics
              </button>
              <button
                class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                type="button"
              >
                Remote
              </button>
              <button
                class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                type="button"
              >
                Work from Home
              </button>
              <button
                class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                type="button"
              >
                Retirement
              </button>
              <button
                class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                type="button"
              >
                Internships
              </button>
              <button
                class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                type="button"
              >
                Freelancer
              </button>
              <button
                class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                type="button"
              >
                Salary and Compensation
              </button>
              <button
                class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                type="button"
              >
                Starting a job
              </button>
            </div>
          </div>
        </div>
      </section>
      <section
        class="w-full flex justify-center bg-white"
      >
        <div
          class="flex flex-col gap-8 sm:gap-[72px] max-w-6xl flex-nowrap  sm:flex-row w-full py-14 sm:py-[60px] px-4"
        >
          <p
            class=" mb-0 text-[34px] leading-10 sm:w-[408px] sm:leading-[60px] sm:text-[50px] font-normal sm:font-light text-color-text-darker"
          >
            Find the right job or internship for you
          </p>
          <div
            class="flex flex-col gap-4 max-w-[343px] sm:max-w-[640px]"
          >
            <p
              class=" uppercase text-color-text font-bold text-[17px]"
            >
              SUGGESTED SEARCHES
            </p>
            <div
              class="flex flex-wrap gap-x-[6px] gap-y-3 text-[19px] sm:text-[21px] font-bold"
            >
              <div>
                <div
                  class="flex flex-wrap gap-x-[6px] gap-y-3 text-[19px] sm:text-[21px] font-bold"
                >
                  <button
                    class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                    type="button"
                  >
                    Engineering
                  </button>
                  <button
                    class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                    type="button"
                  >
                    Business Development
                  </button>
                  <button
                    class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                    type="button"
                  >
                    Finance
                  </button>
                  <button
                    class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                    type="button"
                  >
                    Administrative Assistant
                  </button>
                  <button
                    class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                    type="button"
                  >
                    Retail Associate
                  </button>
                  <button
                    class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                    type="button"
                  >
                    Customer Service
                  </button>
                  <button
                    class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                    type="button"
                  >
                    Operations
                  </button>
                  <button
                    class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                    type="button"
                  >
                    Information Technology
                  </button>
                  <button
                    class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                    type="button"
                  >
                    Marketing
                  </button>
                  <button
                    class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-gray hover:bg-black text-color-button-gray"
                    type="button"
                  >
                    Human Resources
                  </button>
                </div>
                <button
                  aria-expanded="false"
                  aria-label="Show more Suggested Searches"
                  class="flex items-center text-color-text gap-1 mt-1 text-[17px] h-12 w-fit text-left px-2 py-[2px]"
                  type="button"
                >
                  Show more
                  <svg
                    class="pt-[2px]"
                    focusable="false"
                    height="16"
                    preserveAspectRatio="xMinYMin meet"
                    width="16"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M8 9l5.93-4L15 6.54l-6.15 4.2a1.5 1.5 0 01-1.69 0L1 6.54 2.07 5z"
                      fill="currentColor"
                    />
                  </svg>
                </button>
              </div>
            </div>
          </div>
        </div>
      </section>
      <section
        class="w-full flex justify-center bg-[#F1ECE5]"
      >
        <div
          class="flex flex-col gap-8 sm:gap-[72px] max-w-6xl flex-nowrap  sm:flex-row w-full py-14 sm:py-[60px] px-4"
        >
          <p
            class=" mb-0 text-[34px] leading-10 sm:w-[408px] sm:leading-[60px] sm:text-[50px] font-normal sm:font-light text-[#b24020]"
          >
            Post your job for millions of people to see
          </p>
          <div
            class="flex flex-col gap-4 max-w-[343px] sm:max-w-[640px]"
          >
            <p
              class=" uppercase text-color-text font-bold text-[17px]"
            />
            <div
              class="flex flex-wrap gap-x-[6px] gap-y-3 text-[19px] sm:text-[21px] font-bold"
            >
              <button
                class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  hover:bg-opacity-[0.04] border-color-button-blue hover:bg-color-button-blue text-color-button-blue"
                type="button"
              >
                Post a job
              </button>
            </div>
          </div>
        </div>
      </section>
      <section
        class=" min-h-[500px] sm:min-h-[840px] bg-white sm:mt-4 bg-lp-bottom-pattern bg-no-repeat bg-cover sm:bg-auto  w-full flex justify-center bg-[left_top_0.6rem] sm:bg-center"
      >
        <div
          class="max-w-6xl w-full px-4 py-[56px] sm:py-[60px] flex flex-col gap-3 sm:gap-7"
        >
          <h2
            class=" text-color-text-darker text-[34px] sm:text-[56px] sm:leading-[70px] font-extralight"
          >
            Join your colleagues, classmates, and friends on LinkedIn.
          </h2>
          <div
            class=" text-xl"
          >
            <button
              class="flex justify-center items-center py-3 px-6 font-bold rounded-[28px] w-fit border border-solid  border-color-button-blue bg-color-button-blue hover:bg-color-button-blue-darker text-white"
              type="button"
            >
              Get started
            </button>
          </div>
        </div>
      </section>
    </main>
    <footer
      class=" bg-color-blue text-white text-sm  w-full h-12 mt-[-2px] sm:mt-[-8px]"
    >
      <div
        class="flex w-full h-full  items-center justify-center gap-2"
      >
        <span
          class=" dark:text-white"
        >
          Created by 
          <strong>
            <a
              class="dark:text-white hover:underline"
              href="https://github.com/pieroguerrero/linkedin"
              id="link-footer"
              rel="noreferrer"
              target="_blank"
            >
              pieroguerrero
            </a>
          </strong>
        </span>
        <a
          class=" dark:text-white"
          href="https://github.com/pieroguerrero/linkedin"
          rel="noreferrer"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="30px"
            viewBox="0 0 30 30"
            width="30px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M15,3C8.373,3,3,8.373,3,15c0,5.623,3.872,10.328,9.092,11.63C12.036,26.468,12,26.28,12,26.047v-2.051c-0.487,0-1.303,0-1.508,0c-0.821,0-1.551-0.353-1.905-1.009c-0.393-0.729-0.461-1.844-1.435-2.526c-0.289-0.227-0.069-0.486,0.264-0.451c0.615,0.174,1.125,0.596,1.605,1.222c0.478,0.627,0.703,0.769,1.596,0.769c0.433,0,1.081-0.025,1.691-0.121c0.328-0.833,0.895-1.6,1.588-1.962c-3.996-0.411-5.903-2.399-5.903-5.098c0-1.162,0.495-2.286,1.336-3.233C9.053,10.647,8.706,8.73,9.435,8c1.798,0,2.885,1.166,3.146,1.481C13.477,9.174,14.461,9,15.495,9c1.036,0,2.024,0.174,2.922,0.483C18.675,9.17,19.763,8,21.565,8c0.732,0.731,0.381,2.656,0.102,3.594c0.836,0.945,1.328,2.066,1.328,3.226c0,2.697-1.904,4.684-5.894,5.097C18.199,20.49,19,22.1,19,23.313v2.734c0,0.104-0.023,0.179-0.035,0.268C23.641,24.676,27,20.236,27,15C27,8.373,21.627,3,15,3z"
            />
          </svg>
        </a>
      </div>
    </footer>
  </div>
</div>
`;
